client/components/plans/plan-actions/index.jsx

```diff
@@ -251,37 +264,44 @@ module.exports = React.createClass( {
 		analytics.ga.recordEvent( 'Upgrades', 'Clicked Current Plan' );
 	},
 
-	getTrialPlanHint: function() {
-		var remainingDays = this.moment(
+	getTrialPlanHint() {
+		const remainingDays = this.moment(
 				this.props.sitePlan.expiry
-			).diff( this.moment(), 'days' ),
-			translationComponents = {
-				strong: <strong />,
-				link: <a href='#'
-					className="plan-actions__trial-upgrade-now"
-					onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'link' ) } />
-			},
-			hint;
+			).diff( this.moment(), 'days' );
+		let hint, upgradePrompt;
 
 		if ( remainingDays === 0 ) {
-			hint = this.translate(
-				'{{strong}}Your trial ends today.{{/strong}} Like what you see? {{link}}Upgrade Now{{/link}}',
-				{ components: translationComponents }
-			);
+			hint = this.translate( 'Your trial ends today.' );
 		} else {
 			hint = this.translate(
-				'{{strong}}Your trial ends in %(days)d day.{{/strong}} Like what you see? {{link}}Upgrade Now{{/link}}',
-				'{{strong}}Your trial ends in %(days)d days.{{/strong}} Like what you see? {{link}}Upgrade Now{{/link}}',
+				'Your trial ends in %(days)d day.',
+				'Your trial ends in %(days)d days.',
 				{
 					args: { days: remainingDays },
-					count: remainingDays,
-					components: translationComponents
+					count: remainingDays
+				}
+			);
+		}
+
+		if ( config.isEnabled( 'upgrades/checkout' ) ) {
+			hint += ' ';
+			upgradePrompt = this.translate(
+				'Like what you see? {{link}}Upgrade Now{{/link}}',
+				{
+					components: {
+						link: <a href="#"
+							className="plan-actions__trial-upgrade-now"
+							onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'link' ) } />
+					}
 				}
 			);
 		}
 
 		return (
-			<div className="plan-actions__trial-hint">{ hint }</div>
+			<div className="plan-actions__trial-hint">
+				<strong>{ hint }</strong>
+				{ upgradePrompt }
+			</div>
```

Comment:
I removed this method, as we talked above. It's no longer a concern :)
Anyway, good catch. We should keep RTL in mind when combining message translations :+1: